cli/src/cli_util.rs

```diff
@@ -2664,7 +2664,10 @@ impl CliRunner {
         ui: &mut Ui,
         mut layered_configs: LayeredConfigs,
     ) -> Result<(), CommandError> {
-        let cwd = env::current_dir().unwrap(); // TODO: maybe map_err to CommandError?
+        let cwd = env::current_dir().expect(
+            "Could not determine current directory. Did you check-out a commit where the \
+             directory doesn't exist?",
+        ); // TODO: maybe map_err to CommandError?
```

Comment:
Could also map to a CommandError, if that's better?

Reply:
Yes, `user_error()` should be indeed better.

Reply:
Done.